packages/ui/core-components/src/lib/unsorted/viz/table/Column.svelte

```diff
@@ -103,6 +103,12 @@
 	export let chip = false;
 	$: chip = chip === 'true' || chip === true;
 
+	// Sparkline:
+	export let sparklineType = 'line'; // line, area, or bar
+	export let sparklineColor = undefined;
+	export let sparklineDateCol = undefined;
```

Comment:
`"sparklineType" = line` === `sparkLine` ?
`"sparklineType" = bar` === `sparkBar` ?

`sparkColor`
`sparkDate`
`sparkValue`

Reply:
that's a good idea, i like having multiple content types for these